Cargo.toml

```diff
@@ -230,7 +230,8 @@ walkdir = "2.0"
 wasm-bindgen = "0.2.89"
 wasm-bindgen-cli-support = "0.2.89"
 wasm-bindgen-futures = "0.4.33"
-web-sys = "0.3.61"
+# web-sys 0.3.68 is a breaking API change
+web-sys = "=0.3.67"
```

Comment:
We should note _why_ we are locking it (breaking changes in an unstable API used by wgpu)